### ISSUE TITLE
'orderBy' in options doesn't work with two sorting attributes correctly

### ISSUE THREAD

**Issue by ronpfi** [Wakanda/wakanda-issues#233]
WAK 2.3.0 - Serverside:

I have a dataclass called 'Mitglied' which as an storage attribute of type object, called 'Daten'. 'Daten' contains the attributes 'Name' and 'Vorname'.

This code works as expected: the entities are ordered by 'Name' and then by 'Vorname'. 

`var q ='*Müller*';
var ent = ds.Mitglied.query('Daten.Name == :1',q).orderBy('Daten.Name, Daten.Vorname')
`

This code works not as expected: the entities are ordered only by 'Vorname'. Using only one attribute for sorting works, using two - only the second is used. (Using this at client side (angular - client) shows the same behavior).

`var q = '*Müller*';
var ent = ds.Mitglied.query('Daten.Name == :1', q, {
    orderBy: 'Daten.Name, Daten.Vorname'
})`

**Comment by liuxiang625** [comment]
Can you post and compare the REST requests generated by both `query('Daten.Name == :1',q).orderBy('Daten.Name, Daten.Vorname')` and `query('Daten.Name == :1', q, { orderBy: 'Daten.Name, Daten.Vorname' })`? 
The orderby parameter should look like: `"$orderby="Daten.Name, Daten.Vorname"`

**Comment by ronpfi** [comment]
Client side with orderBy as an extension of the query produces this REST request (which sort all only by the second attribute 'Daten.Vorname':
`* Request URL:http://localhost:4201/rest/Mitglied?$filter=%22Daten.Name%20==%20:1%20||%20Daten.Vorname%20==%20:1%20||%20Daten.Personen[].Vorname%20==%20:1%20||%20Daten.Personen[].Nachname%20==%20:1%20||%20Daten.Strasse%20==%20:1%20||%20Daten.Personen[].Kartennummer%20==%20:1%20||%20Daten.Mitgliedsnummer%20==%20:1%22&$orderby=Daten.Name%20ASC,%20Daten.Vorname%20ASC&$params=%27[%22*M%C3%BCller*%22]%27&$limit=5000&$method=entityset`

**Comment by vegarringdal** [comment]
@ronpfi usually easier for someone to read/find bugs if you copy the image from the request

Something like:

![image](https://user-images.githubusercontent.com/2901416/34576430-2f00695e-f17e-11e7-839e-96d00bf7bc1e.png)

**Comment by JegoYann** [comment]
Helloe @ronpfi 

The second syntax is not standard and is probably not working in Wakanda. I can't find anything on it in the documentation.

Here is the two standard syntaxes:

```
ds.employee.query('city == "Chicago"').orderBy('name desc');
ds.employee.query('city == "Chicago" order by name desc');
```

**Comment by JegoYann** [comment]
Hello @ronpfi ,

This issue was fixed with v2.4.0.

Please, do not hesitate to reopen this issue if you need it.
